### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Inappropriate control type for the Table in "Deploy web app dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1396]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app.
4. "Deploy web app" dialog will open. 
5. In "Deploy web app" dialog, tab to table and observe the control type using inspect

**Actual Result:**
Control type for table is list.
Narrator table commands such as "Caps+F3", "Caps+F8" are working fine.

**Narrator :**
Tab
webapp-180112171944, <say-as interpret-as="spell">2</say-as> of 15, selected, 

**NVDA:**
list
webapp-180112171944; JDK: 1.8; Web container: TOMCAT 8.5; Resource group: adobe-rg  2 of 15

**Expected Result:**
Control type should be Table.
Narrator and NVDA should announce as
Table
webapp-180112171944, <say-as interpret-as="spell">2</say-as> of 15, selected, 

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![deploy web app-table-inspect](https://user-images.githubusercontent.com/22956130/35683832-2e07d71c-078b-11e8-8495-5ab9c312d306.jpg)
![deploy web app-table-narrator](https://user-images.githubusercontent.com/22956130/35683837-3160f9a2-078b-11e8-9a49-f3442563d396.jpg)
![deploy web app-table-nvda](https://user-images.githubusercontent.com/22956130/35683844-33694786-078b-11e8-900e-471ceab785f9.jpg)

**Comment by Eskibear** [comment]
duplicated with #1405

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749841
